.drone.yml

```diff
@@ -7,6 +7,21 @@ platform:
   arch: amd64
 
 steps:
+- name: jenkins_test_image_build
+  image: docker
+  commands:
+  - cd jenkins_test_setup
+  - docker build -t jenkins_test_setup:latest .
+  volumes:
```

Comment:
not sure if this will help but drone seems to complain about mounting a volume, so maybe you can avoid that by using DinD 
https://discourse.drone.io/t/build-and-test-docker-images-with-dind/1808